crates/red_knot_python_semantic/src/types.rs

```diff
@@ -2664,6 +2699,10 @@ impl<'db> Type<'db> {
             Type::Callable(CallableType::WrapperDescriptorDunderGet) => {
                 KnownClass::WrapperDescriptorType.to_class_literal(db)
             }
+            Type::Callable(CallableType::General(_)) => {
+                // TODO
```

Comment:
Can we add a small explanation of what needs to be done here or remove the comment if it's redundant to the todo_type description

Reply:
I'll expand the comment. I avoided here as it would be similar to `todo_type` but I still grep for `TODO` comments 😅